### PR TITLE
investigate, update data +viz; fix css; adjust color accessibility; c…

### PR DESCRIPTION
**Pull request by NotLiu**
…ontent changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,8 +30,8 @@ <h3 class="explainer__sub-heading">Here is what we found</h3>
   <div class="explainer__right">
     <h2 class="explainer__heading">Getting Parking Right Matters</h2>
     <div class="explainer__center">
-      <img class="explainer__image" src="./assets/images/explainer-before.png">
-      <img class="explainer__image" src="./assets/images/explainer-after.png">
+      <img class="explainer__image" src="./assets/images/explainer-before.png" alt="Before Image of urban design with parking garage, cars, and buildings">
```
Comment (anaximander) [inline index.html:33]: Good accessibility improvement 👍

**Review by anaximander — CHANGES_REQUESTED** [review]
Made note of a couple things that need to be removed before we merge this.

On staging, I'm seeing some "NaN" values in the "Parking Supply per Unit" column (appears to be related to the new data). Similarly, there are quite a few 0 values in the "Jobs Accessible by Transit" column for the new data: is this data missing in the new dataset? Might need to follow up with Adi/Conor to get that squared away. If we just don't have the numbers, I think updating that table to display "N/A" or "--" or something like that would convey that information more correctly than a 0.

File: assets/javascripts/parking_site_map.js [hunk 2]
```diff
@@ -101,14 +112,19 @@ function createSlider(sliderData) {
...
-  const g = sliderSvg.append('g').attr('transform', `translate(${sliderMargin.left}, ${sliderMargin.top})`);
+  const g = sliderSvg.append('g').attr('transform', `translate(${sliderMargin.left}, ${sliderMargin.top})`).attr("class", "slider-group");
 
   const sliderX = d3.scaleLinear().domain(d3.extent(park_dem)).range([0, sliderWidth]);
   const sliderY = d3.scaleLinear().domain(d3.extent(utilization)).range([0, sliderHeight]);
 
+  const y_axis = d3.axisLeft().scale(sliderY).tickFormat(d3.format("~%"));
+  const maxX = d3.max(park_dem);
+  const minX = d3.min(park_dem);
+  console.log(minX);
```
Comment (anaximander) [inline assets/javascripts/parking_site_map.js:126]: Please remove this before merge

File: assets/javascripts/parking_site_map.js [hunk 3]
```diff
@@ -157,7 +177,32 @@ function createSlider(sliderData) {
       .startAngle(0)
       .endAngle((d, i) => { return i ? Math.PI : -Math.PI; }));
 
-  gBrush.call(brush.move, [0.3, 0.5].map(sliderX));
+  console.log(brushState);
```
Comment (anaximander) [inline assets/javascripts/parking_site_map.js:180]: Please remove this before merge

File: assets/javascripts/parking_site_map.js [hunk 4]
```diff
@@ -260,24 +305,42 @@ function createTable(data) {
...
     .append('tr')
-    .attr('class', 'parking-table__data-row')
+    .attr('class', "parking-table__data-row")
     .attr('id', d => `block-${d.site_id}`)
     .on('click', d => toggleSelected(d));
   rows.selectAll('td')
     .data(row => [row.prop_name,
       row.muni,
-      parseFloat(row.park_sup_tot).toFixed(2),
+      row.park_sup_tot != "" ? parseFloat(row.park_sup_tot).toFixed(2) : "--",
```
Comment (anaximander) [inline assets/javascripts/parking_site_map.js:312]: 👍

File: assets/javascripts/parking_site_map.js [hunk 5]
```diff
@@ -260,24 +305,42 @@ function createTable(data) {
...
       parseFloat(row.park_dem).toFixed(2),
       Math.round(+row.util_rate * 100),
       Math.round(+row.bldg_affp * 100),
       row.walk_score,
-      Number(row.b_umn_t30jobs).toLocaleString()])
+      row.b_umn_t30jobs != "" ? Number(row.b_umn_t30jobs) : "--"])
```
Comment (anaximander) [inline assets/javascripts/parking_site_map.js:317]: Thanks for making this change, I think it clarifies things for the user 👍

File: assets/javascripts/parking_site_map.js [hunk 6]
```diff
@@ -321,16 +399,105 @@ window.addEventListener('DOMContentLoaded', () => {
...
+      'LYNN',
+      'STONEHAM',
+      'WOBURN',
+      'WELLESLEY',
+      'SWAMPSCOTT',
+      'MARBLEHEAD',
+      'NAHANT',
+      'WINCHESTER'
+    ];
+    phases = ["1", "2", "3", "4"];
```
Comment (anaximander) [inline assets/javascripts/parking_site_map.js:424]: Not anything to block a merge/deploy, but we could consider using a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) here. Not sure if order matters (maybe it does, skimming the rest of the code below), but if not, it's faster/easier to use `includes` with Set types. Probably easier to add/remove members of the set vs. Array splicing.

File: index.html [hunk 7]
```diff
@@ -66,15 +82,30 @@ <h2 class="explore-the-data__header">Adjust the Slider to Explore the Data</h2>
...
+      </div>
     </div>
   </div>
 </section>
 <section class="action-boxes">
   <div class="action-box">
-    <h2 class="action-box__title">Read the Full Report</h2>
-    <p class="action-box__text">Find the full report and technical appendix here.</p>
+    <h2 class="action-box__title">Read the Full Report (Phases 1 and 2)</h2>
+    <p class="action-box__text">Find the full phase 1 & 2 report and technical appendix here.</p>
```
Comment (anaximander) [inline index.html:106]: Not a blocker, but we could capitalize "phases" -> "Phases" to be consistent with other usage.

File: assets/javascripts/parking_site_map.js (hunk 6 above)
Comment (NotLiu) [reply assets/javascripts/parking_site_map.js:424]: Good point, it should be a quick fix. Let me do that right now

**Review by anaximander — APPROVED** [review]

**Comment by anaximander** [comment]
Looks good to me! 👍 Nice work!

**Pull request merged**